sway-lsp/src/capabilities/hover.rs

```diff
@@ -10,7 +10,9 @@ use crate::{
     },
 };
 use sway_core::{
-    declaration_engine, semantic_analysis::ast_node::TypedDeclaration, Declaration, Visibility,
+    declaration_engine::{self, de_get_struct},
```

Comment:
Change `declaration_engine::{self, de_get_struct},` to `declaration_engine`, and change references to `de_get_struct` into `declaration_engine::de_get_struct`. This is done because `sway-lsp` is a different crate.